src/main/resources/assets/structurize/lang/en_us.json

```diff
@@ -153,7 +153,7 @@
   "block.structurize.gated_acacia_paper_timber_frame": "Gate Framed Acacia Paper Timber Frame",
   "block.structurize.up_gated_jungle_dark_oak_timber_frame": "Up Gate Framed Jungle Dark Oak Timber Frame",
   "block.structurize.blockbarreldeco_onside": "Barrel Horizontal",
-  "block.structurize.birch_blockpaperwall": "Birch Paperwall",
```

Comment:
I would say this is generated by datagen

Reply:
The name of the item is generated by datagen? Can it still be changed?

Reply:
Blame Ash, he hasn't made paper walls datagen. But yes, timber frames translations are generated

Reply:
Good thing I didn't change any of the timber frame names then